### PR TITLE
Lagom: Mention OSS-Fuzz in ReadMe

### PR DESCRIPTION
**Pull request by nico**
We added OSS-Fuzz integration in #4154, but documentation about it
is spread across several pull requests, IRC, and issues. Let's collect
the important bits in the ReadMe.

### CODE REVIEW

**Comment by nico** [comment]
@linusg @DavidKorczynski @BenWiederhake @Lubrsi @bugaevc

File: Meta/Lagom/ReadMe.md [hunk 1]
```diff
@@ -30,6 +34,32 @@ To run several fuzz jobs in parallel, pass `-jobs=24 -workers=24`.
...
 It's good to move overzealous log output behind `FOO_DEBUG` macros.
 
+### Fuzzing on OSS-Fuzz
+
+https://oss-fuzz.com/ automatically runs all fuzzers in the Fuzzers/ subdirectory whose name starts with "Fuzz" and which are added to the build in `Fuzzers/CMakeLists.txt` if `ENABLE_OSS_FUZZ` is set. Looking for "serenity" on oss-fuzz.com finds interesting links, in particular:
+
+* [known open bugs found by fuzzers](https://oss-fuzz.com/testcases?project=serenity&open=yes)
+  * [oss-fuzz bug tracker for these](https://bugs.chromium.org/p/oss-fuzz/issues/list?sort=-opened&can=1&q=proj:serenity)
+* [coverage report](https://oss-fuzz.com/coverage-report/job/libfuzzer_asan_serenity/latest)
+* [build status](https://oss-fuzz-build-logs.storage.googleapis.com/index.html#serenity)
```
Comment (linusg) [inline Meta/Lagom/ReadMe.md:44]:
```suggestion
* [build logs](https://oss-fuzz-build-logs.storage.googleapis.com/index.html#serenity)
```

**Review by linusg — APPROVED** [review]
Great idea, LGTM!

**Comment by BenWiederhake** [comment]
I suggest that we try to keep the actual testcases out of the main serenity repo, that's why I (helped in) creating https://github.com/SerenityOS/serenity-fuzz-corpora .

Currently, the `bmp_suite` ist technically GPL, but not really marked as such. That's fine as long as the bmp suite is clearly separated from "our" test cases. But the jpg fuzz testcases really should not be mixed with the GPL testsuite, just to make sure we never get into any hot water there.

Do you want to add a sentence in there, or should I?

**Comment by nico** [comment]
@BenWiederhake Let's do that in a follow-up :)

**Pull request merged**